app/lib/routes/settings.dart

```diff
@@ -119,6 +229,25 @@ class BellSoundControl extends StatelessWidget {
   }
 }
 
+class ShareBikeControl extends StatelessWidget {
+  const ShareBikeControl(this.bike, {super.key});
+
+  final Bike bike;
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: () {
+        showModalBottomSheet(
+          context: context,
+          builder: (context) => ShareSettings(bike: bike),
+        );
+      },
```

Comment:
This can be simplified
```suggestion
      onPressed: () =>
        showModalBottomSheet(
          context: context,
          builder: (context) => ShareSettings(bike: bike),
        ),
```